env.go

```diff
@@ -0,0 +1,95 @@
+package envconf
+
+import (
+	"bufio"
+	"bytes"
+	"errors"
+	"io"
+	"os"
+	"strings"
+)
+
+var (
+	ErrInvalidPair = errors.New("invalid pair for env variable")
+)
+
+const (
+	commentSymbol = '#'
```

Comment:
can be defined in Parse func scope

Reply:
@antonmashko I think this comment relates more to the style of the code.

Reply:
yep, it is